intellij-plugin-verifier/verifier-test/src/test/java/com/jetbrains/pluginverifier/tests/bytecode/Dumps.kt

```diff
@@ -0,0 +1,342 @@
+package com.jetbrains.pluginverifier.tests.bytecode
+
+import org.objectweb.asm.AnnotationVisitor
+import org.objectweb.asm.ClassWriter
+import org.objectweb.asm.FieldVisitor
+import org.objectweb.asm.Label
+import org.objectweb.asm.MethodVisitor
+import org.objectweb.asm.Opcodes
+import org.objectweb.asm.Opcodes.*
+
+
+object Dumps {
```

Comment:
add test sources and compiled`.jar` file to testdata folder containing relevant cases instead of manually outputting?

Reply:
actually standalone bytecode is easier to maintain